### PR TITLE
Allow tabbing to vertical menu items while nudge is displayed

### PR DESCRIPTION
**Pull request by RoyEJohnson**
For: https://github.com/openstax/unified/issues/2691
Added fix for: https://github.com/openstax/unified/issues/2703

### CODE REVIEW

**Review by TomWoodward — APPROVED** [review]

File: src/app/content/components/NudgeStudyTools/index.tsx [hunk 1]
```diff
@@ -119,19 +120,40 @@ const NudgeStudyTools = ({
 
 function useTabNavigationInterceptor() {
   const ref = React.useRef<HTMLElement>();
+  const isMobile = useMatchMobileMediumQuery();
 
   React.useEffect(
     () => {
-      const abortTabbing = (event: Event) => {
+      if (isMobile) {
```
Comment (jivey) [inline src/app/content/components/NudgeStudyTools/index.tsx:127]: Does this need to exclude mobile? I'm thinking external keyboard, iPad narrow window view, etc.

Comment (RoyEJohnson) [reply src/app/content/components/NudgeStudyTools/index.tsx:127]: Looks like keyboard navigation does need to be supported for mobile.
https://bootcamp.uxdesign.cc/keyboard-accessibility-mobile-devices-688dd73bdb7b

Comment (RoyEJohnson) [reply src/app/content/components/NudgeStudyTools/index.tsx:127]: The bad news is that I don't get keydown events from the browser when in responsive mode. :(

Comment (jivey) [reply src/app/content/components/NudgeStudyTools/index.tsx:127]: I wonder if the xcode ios simulator would work for testing this 🤔

Comment (RoyEJohnson) [reply src/app/content/components/NudgeStudyTools/index.tsx:127]: It got merged, so maybe we'll want a new issue for the mobile.

**Pull request merged**